src/objects/strategy-cards/register-standard-card.js

```diff
@@ -0,0 +1,65 @@
+const {
+    broadcastMessage,
+    onUiClosedClicked,
+    registerStrategyCard,
+} = require("./strategy-card");
+const { Button, Text, VerticalBox } = require("../../wrapper/api");
+const locale = require("../../lib/locale");
+
+module.exports = function registerStandardCard(refObject, name, color) {
```

Comment:
I'd rename the parameter "object" or "strategyCardObject", leave refObject to mean only the api-given value.

Reply:
wish granted in #71